File: Lemur/src/frc/team4330/robot/commands/StartClimb.java
package frc.team4330.robot.commands;

import edu.wpi.first.wpilibj.command.Command;
import frc.team4330.robot.Robot;
import frc.team4330.robot.RobotMap;

public class StartClimb extends Command {

	public StartClimb() {
		requires(Robot.tarzan);
	}
	@Override
	protected void initialize() {
		Robot.tarzan.set(RobotMap.REEL_SPEED);
	}
	
	
	@Override
	protected boolean isFinished() {
		return false;
	}

	@Override
	protected void end() {
		Robot.tarzan.stop();
	}
	
	@Override
	protected void interrupted() {
		System.out.println(this.getName() + "was interrupted!");
		end();
	}
}

File: Lemur/src/main/java/frc/team4330/robot/commands/StartClimb.kt
package frc.team4330.robot.commands

import edu.wpi.first.wpilibj.command.Command
import frc.team4330.robot.Robot
import frc.team4330.robot.RobotMap

class StartClimb : Command() {
    init {
        requires(Robot.tarzan)
    }

    override fun initialize() {
        Robot.tarzan.set(RobotMap.REEL_SPEED)
    }


    override fun isFinished(): Boolean {
        return false
    }

    override fun end() {
        Robot.tarzan.stop()
    }

    override fun interrupted() {
        println(this.name + "was interrupted!")
        end()
    }
}
